app/views/users/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Users</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @users.each do |user| %>
+      <tr>
+        <td><%= user.name %></td>
+        <td><%= link_to 'Show', user %></td>
+      </tr>
+    <% end %>
```

Comment:
If you create a `_user.html.erb` partial you can call it here using 
  ```
<tbody>
    <%= render @users %>
  </tbody>
```
It's magical.